src/utils/self_test/SConscript

```diff
@@ -13,12 +13,13 @@ SELF_TEST = 'self_test.c'
 def scons():
     """scons function"""
 
-    Import('env', 'prereqs', 'cart_lib', 'gurt_lib', 'cart_utils_objs')
+    Import('env', 'prereqs', 'cart_lib', 'gurt_lib', 'cart_utils_objs',
+           'dc_mgmt_tgts')
 
     tenv = env.Clone(LIBS=[])
 
-    tenv.AppendUnique(LIBS=['cart', 'gurt', 'pthread', 'm'])
-    prereqs.require(tenv, 'mercury')
+    tenv.AppendUnique(LIBS=['cart', 'gurt', 'pthread', 'm', 'daos'])
+    prereqs.require(tenv, 'mercury', 'protobufc')
```

Comment:
(pylint-undefined-variable) Undefined variable 'prereqs'

Reply:
`prereqs` was not added in this PR, though it seems pylint may not account for how we're [exporting `prereqs` in SConstruct](https://github.com/daos-stack/daos/blob/master/SConstruct#L383) (see [SCons Import/Export docs](https://scons.org/doc/2.0.1/HTML/scons-user.html#AEN3295)).  Can we pylint disable this message?